packages/components/organisms/f-footer/src/components/LinkList.vue

```diff
@@ -88,32 +86,51 @@ export default {
             return this.currentScreenWidth <= 1024;
         }
     },
-
     mounted () {
         this.currentScreenWidth = windowServices.getWindowWidth();
         windowServices.addEvent('resize', this.onResize, 100);
 
-        if (this.isBelowWide) {
-            this.panelCollapsed = true;
-        }
+        this.setPanelCollapsed();
     },
-
     destroyed () {
         windowServices.removeEvent('resize', this.onResize);
     },
-
     methods: {
+        /**
+         * Sets Links List panel collapsed state.
+         * Affects `is-collapsed` panel class.
+         * Value for below `wide` screen width is based on saved `interactedState`, intially defaults to `true` - collapsed.
+         * Value for wider screens resets to `false` e.g. expanded Links List. (see `onResize`)
+         */
+        setPanelCollapsed () {
+            if (this.isBelowWide) {
+                this.panelCollapsed = this.interactedState === null ? true : this.interactedState;
+            } else {
+                this.panelCollapsed = false;
+            }
+        },
+        /**
+         * Handle click events on Link List visibility toggle.
+         * Only applied to below `wide` screen width (ref. Fozzie UI breakpoints).
+         */
         onPanelClick () {
             if (this.isBelowWide) {
-                this.panelCollapsed = !this.panelCollapsed;
+                this.interactedState = !this.panelCollapsed;
+                this.setPanelCollapsed();
             }
         },
-
+        /**
+         * Handles `resize` window events.
+         * Screen width is the only factor that affects Links List presentation.
+         * Note: Devices trigger `resize` on scroll when address bar collapses
+         */
         onResize () {
-            this.currentScreenWidth = windowServices.getWindowWidth();
+            const newScreenWidth = windowServices.getWindowWidth();
 
-            if (this.isBelowWide) {
-                this.panelCollapsed = true;
+            if (this.currentScreenWidth !== newScreenWidth) {
+                this.currentScreenWidth = newScreenWidth;
+
+                this.setPanelCollapsed();
```

Comment:
Is it worth throttling this event for manual resize on desktop?

Reply:
This is already throttled where the `resize` event listener is added, line [91](https://github.com/justeat/fozzie-components/pull/770/files/37b19880de927d8efcf73fb46e8ba738aa565bb1#diff-f86486deb63b13c5ddc77cfb17849fc0fbdb781766f2db38652de9cc135535c8R91) 
The other uses are based on user interaction.